lib/r10k/git/shellgit/working_repository.rb

```diff
@@ -49,14 +49,14 @@ def clone(remote, opts = {})
   # @param ref [String] The git reference to check out
   # @param opts [Hash] Optional hash of additional options.
   def checkout(ref, opts = {})
+    options = ['checkout', ref]
```

Comment:
could you name this variable `argv` to be consistent with the `clone` method above it?